spec/rubocop/cop/lint/redundant_type_conversion_spec.rb

```diff
@@ -294,5 +298,9 @@
     it_behaves_like 'offense', :to_set, '::Set.new([1, 2, 3])'
     it_behaves_like 'offense', :to_set, 'Set[1, 2, 3]'
     it_behaves_like 'offense', :to_set, '::Set[1, 2, 3]'
+
+    it_behaves_like 'accepted', 'Set.new(set).to_set { |item| foo(item) }'
```

Comment:
Same here.

Reply:
Since `Set` doesn't have a literal syntax, it has been replaced with `Set[...]`, which is the closest in meaning, to match the existing example.